core/domain/email_manager_test.py

```diff
@@ -2928,7 +2928,7 @@ def test_send_removed_voiceover_reviewer_email(self):
             'in the Hindi language. You won\'t be able to review voiceover '
             'applications made by contributors in the Hindi language any more, '
             'but you can still contribute voiceovers through the '
-            '<a href="https://www.oppia.org/community_dashboard">'
+            '<a href="https://www.oppia.org/community-dashboard">'
```

Comment:
Hm, I think the communty dashboard can be accessed without logging in. In that case, should we be redirecting the old URL to this too?

Also, can you point me to the changes to creator / learner dashboard.

Reply:
And topics and skills dashboard.

Reply:
> Hm, I think the communty dashboard can be accessed without logging in. In that case, should we be redirecting the old URL to this too?

Done.

The commit for changes are:
[creator-dashboard](https://github.com/oppia/oppia/pull/9460/commits/bedd707aabe926b109692d6c3a2f507d39e3a4b5)
[learner-dashboard](https://github.com/oppia/oppia/pull/9460/commits/e821291f75f0b37eb65671f9322d6d77005a439d)
[topics-and-skills-dashboard](https://github.com/oppia/oppia/pull/9460/commits/00b89a507f365b8e434e2ddcbc8cec02c4dbf120)